src/pool/srv_util.c

```diff
@@ -322,7 +322,7 @@ update_one_tgt(struct pool_map *map, struct pool_target *target,
 				target->ta_comp.co_rank,
 				target->ta_comp.co_index, map);
 			target->ta_comp.co_status = PO_COMP_ST_UP;
-			++(*version);
+			target->ta_comp.co_fseq = ++(*version);
```

Comment:
@wangdi1, @byron-marohn, could you review this one-line change, please?

Reply:
Not sure, but this does seem potentially problematic to me. This will mean that rebuild loses track of the original failure sequence for this device, which might be needed during reintegration. It's possible this is a needed bugfix, but this seems like a strange patch to include it in. 

Am fine with this if the rebuild / reintegration tests actually ran on this and worked better or at least the same... 

@kccain can you explain why this change was made? Was this a bug you found during testing and fixed?

Reply:
By some happenstance this exact same piece of code turned up in a different conversation today - and we tried it out and indeed this change is incorrect and will cause reintegration problems. Definitely needs to be reverted to agree with master.